Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -50,6 +50,56 @@ Building a reverse engineering environment for free is possible. However, there
 
 Because Objective-C and Swift are fundamentally different, the programming language in which the app is written affects the possibilities for reverse engineering it. For example, Objective-C allows method invocations to be changed at runtime. This makes hooking into other app functions (a technique heavily used by [Cycript](http://www.cycript.org/ "Cycript") and other reverse engineering tools) easy. This "method swizzling" is not implemented the same way in Swift, and the difference makes the technique harder to execute with Swift than with Objective-C.
 
+In iOS, all the application code, for both Swift and Objective-C, is compiled to native, unlike Android where code is compiled into Dalvik Bytecode and native code (if code written in C/C++). Thus, to analyze iOS application a disassembler is needed. 
+
+Before getting into how to disassemble an application's code, it is important to know that the application downloaded via Apple Appstore is encrypted using Fairplay DRM and need to be decrypted before it can be analyzed. This has been already discussed in "[Acquiring the App Binary](0x06b-basic-security-testing.md#acquiring-the-app-binary "Acquiring the App Binary")" section. In this section we assume you already have access to a decrypted iOS application.
```

Comment:
For the overal text, try to write in the active voice and try to refer to chapter sections as part of a chapter for a better ebook/printed book :
```suggestion
If you want to disassemble an application's code from the Apple Appstore, remove the Fairplay DRM first. Section "[Acquiring the App Binary](0x06b-basic-security-testing.md#acquiring-the-app-binary "Acquiring the App Binary")" in the chapter "iOS Basic Security Testing" explains how. 
```